doc/tutorials/mount.md

```diff
@@ -1,56 +1,146 @@
 
-- [ ] Understanding Backends
-- [ ] Understanding Mounting
-- [ ] Understanding Plugins
-- [ ] Understanding get/set
+The heart of the approach is the so called _mounting_ of configuration files into the key database.
 
-## Limitations
+Let us start with a motivating example first:
 
-The approach is not ideal when the application itself also changes the
-configuration thus Elektra is bypassed then.  But th Furthermore one
-cannot easily change the configuration format file format (it must be
-in the format the application understands) thus one loses quite some
-flexibility.
+## Mount the lookup table for hostnames
 
-But many applications are only reading the configuration file and for
-many setups it is acceptable not being able to change the file format.
-The heart of the approach is the so called mounting of configuration files
-into the key database.  Note that mounting requires root permissions.
+We mount the lookup table with the following command:
 
     sudo kdb mount /etc/hosts system/hosts hosts
 
-1. The first argument is the configuration file we want to mount, 2. the
-second which path it should have in the key database, and 3. the third
-is the plugin that can read and write this configuration format.
+1. `/etc/hosts` is the configuration file we want to mount
+2. `system/hosts` is the path it should have in the key database, also known as **mountpoint**
+3. `hosts` is the _storage plugin_ that can read and write this configuration format.
 
-We can verify that the file is mounted via:
+Without arguments `kdb mount` lists the currently mounted backends.
 
-    kdb file system/hosts
+    $ kdb mount
+    /etc/hosts on system/hosts with name system/hosts
 
-After mounting a file, we can modify the keys below `system/hosts`.
-We need again to be root, because we modify `/etc/hosts`.
+Which verifies, that all configuration below `system/hosts` is stored in /etc/hosts.
+
+After mounting a file, we can modify keys below `system/hosts`.
+We need to be root, because we modify `/etc/hosts`.
 
     sudo kdb set system/hosts/ipv4/mylocalhost 127.0.0.33
 
-These changes are automatically reflected in `/etc/hosts`:
+These changes are reflected in `/etc/hosts` instantly:
 
-    cat /etc/hosts 127.0.0.33      mylocalhost 127.0.0.1       localhost
+    $ cat /etc/hosts
+    ...
+    mylocalhost 127.0.0.33       localhost
     ...
 
-Thus we wrote using the format used by all applications, applications
-will now pick up these changes:
+Applications will now pick up these changes:
+
+    $ ping mylocalhost
+
+###### Why you need superuser privileges to mount files ######
+Elektra manages its mountpoints in configuration below **system/elektra/mountpoints**.
+The file that holds this configuration is per default only writable by administrators:
+
+    $ kdb file system/elektra/mountpoints
+    /etc/kdb/elektra.ecf
+
+Because of that only root users can mount files.
+
+> At the moment a workaround for this is to add your user
```

Comment:
I would not give workaround tips here.

This workaround might be important for Elektra developers (and should be added in some "Getting Started" for developers) but it is certainly not a tip for everyone. Admins should use `sudo` thus they actually modify system configuration. The tutorial should try to explain it.